uwblueprint/sistering
# Create volunteer shifts table

## Ticket link
Closes #122
Closes #121 



## Implementation description
* Created `VolunteerShiftsTableRow`, `VolunteerShiftsTable` and `VolunteerShiftsTableDate` to display volunteer shifts
* Added theme variants to the `Link` and `Table` components



## Steps to test
(Note: mock data is hardcoded right now under `VolunteerShiftsPage`)
1. Navigate to `/volunteer/shifts` on http://localhost:3000/ 
2. Verify that table data is rendered correctly for the two variations of table data 
3. In the `VolunteerShiftsPage` component, edit the `shifts` prop for `VolunteerShiftsTable` to be empty array. Verify that the empty state for the table component is rendered correctly. 



## What should reviewers focus on?
* Correct behaviour for the table 
* Code logic


## Checklist
